index.html

```diff
@@ -46,46 +46,257 @@ <h3 class="subtitle">
+              />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img
+                class="social-media-icon"
+                src="images/social/angelist.png"
+                alt="angelist"
+              />
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://twitter.com/Kimutai_01?t=oB6Cj93u_SKHZdHeoomTpg&s=09"
+            >
+              <img
+                class="social-media-icon"
+                src="images/social/twitter.png"
+                alt="twitter"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img
+                class="social-media-icon"
+                src="images/social/medium.png"
+                alt="medium"
+              />
+            </a>
+          </li>
+        </ul>
+      </div>
+      <div class="illustration">
         <img
-          class="social-media-icon"
-          src="images/social/angelist.png"
-          alt="angelist"
+          src="images/illustration/illustration-header-mobile.png"
+          alt="illustrations"
         />
-      </a></li>
-        <li><a href="https://twitter.com/Kimutai_01?t=oB6Cj93u_SKHZdHeoomTpg&s=09">
-          <img
-            class="social-media-icon"
-            src="images/social/twitter.png"
-            alt="twitter"
-          />
-        </a></li>
-        <li><a href="#">
-          <img
-            class="social-media-icon"
-            src="images/social/medium.png"
-            alt="medium"
-          />
-        </a></li>
-      </ul>
       </div>
-      <div class="illustration">
-        <img src="images/illustration/illustration-header-mobile.png" alt="illustrations" />
+    </section>
+
+    <section class="works">
+      <h3 class="head">My Recent Works</h3>
+      <div class="separation">
+        <img src="images/Vector.png" alt="line">
+      </div>
+      <div class="card single">
+        <div class="single-work">
+          <img src="images/gym.png" alt="gym" />
+        </div>
+        <h3 class="project-title">Multi-Post stories</h3>
+        <p class="describe">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <div class="main-tags">
+          <div>
+            <a href="#"><img src="images/tags/Tag.png" alt="" /></a>
+          </div>
+          <div>
+            <a href="#"><img src="images/tags/Tag1.png" alt="" /></a>
+          </div>
+          <div>
+            <a href="#"><img src="images/tags/Tag2.png" alt="" /></a>
+          </div>
+          <div>
+            <a href="#"><img src="images/tags/Tag3.png" alt="" /></a>
+          </div>
+  
+        </div>
+          <img class= 'main-butt' src="images/Enable.png" alt="button">
+      </div>
```

Comment:
- [ ] Kindly put the list of project technologies are a list `ul` tag and each of them in a `li` tag
- [ ] Please The button "See project" is a link with the `a` tag or `button` tag, and follow the button styling: Active, Hover, and Pressed.